woocommerce/woocommerce-gutenberg-products-block
# Add featured category block preview

Based on the featured product block preview, adds one to the featured category block.

Closes #1077

### Screenshots

![Screenshot 2019-11-04 at 11 21 02](https://user-images.githubusercontent.com/90977/68117517-72531180-fef5-11e9-9cae-1bc4cb8c1d27.png)

### How to test the changes in this Pull Request:

1. Ensure preview is visible when adding block
2. Ensure once added to the editor the block continues to work without err
